python/cudf/cudf/tests/indexes/datetimeindex/methods/test_tz_convert.py

```diff
@@ -14,3 +15,8 @@ def test_tz_convert():
     pidx = pidx.tz_localize("UTC")
     idx = idx.tz_localize("UTC")
     assert_eq(pidx.tz_convert(tz), idx.tz_convert(tz))
+
+
+def test_tz_convert_naive_typeerror():
+    with pytest.raises(TypeError):
```

Comment:
Would be nice to match the error
```suggestion
    with pytest.raises(TypeError, match=...):
```

Reply:
Can do throughout where it makes sense to